src/functionalTest/java/uk/gov/hmcts/darts/usermanagement/UserManagementFunctionalTest.java

```diff
@@ -0,0 +1,102 @@
+package uk.gov.hmcts.darts.usermanagement;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.json.JSONObject;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.Customization;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.skyscreamer.jsonassert.RegularExpressionValueMatcher;
+import org.skyscreamer.jsonassert.comparator.CustomComparator;
+import uk.gov.hmcts.darts.FunctionalTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class UserManagementFunctionalTest extends FunctionalTest {
+
+    @AfterEach
+    void tearDown() {
+        clean();
+    }
+
+    @Test
+    void shouldCreateUser() {
+        Response response = createUser();
+
+        JSONAssert.assertEquals(
+            """
+                {
+                    "id": "",
+                    "full_name": "James Smith",
+                    "email_address": "james.smith@hmcts.net",
+                    "description": "Functional test user",
+                    "state": "ENABLED",
+                    "security_groups": [ ]
+                }
+                """,
+            response.asString(),
+            new CustomComparator(
+                JSONCompareMode.NON_EXTENSIBLE,
+                new Customization("id", new RegularExpressionValueMatcher<>("\\d*"))
```

Comment:
I guess this regex should really be `\\d+` 
+ matches the previous token between one and unlimited times, as many times as possible

Reply:
Well spotted, updated.